api/dvmodel.proto

```diff
@@ -97,6 +97,7 @@ message Task {
   int32 num_comments = 18 [(gogoproto.moretags) = "quad:\"schema:numComments,optional\""];
   int32 num_upvotes = 19 [(gogoproto.moretags) = "quad:\"schema:numUpvotes,optional\""];
   int32 num_downvotes = 20 [(gogoproto.moretags) = "quad:\"schema:numDownvotes,optional\""];
+  string estimated_time = 21 [(gogoproto.moretags) = "quad:\"schema:estimatedTime,optional\""];
```

Comment:
```suggestion
  string estimated_duration = 21 [(gogoproto.moretags) = "quad:\"schema:estimatedDuration,optional\""];
```

Or maybe you have an even more accurate name?

Reply:
Indeed, estimated_duration is more suitable!